### PR TITLE
Tutorial for triggers by scripting added

### PR DESCRIPTION
**Pull request by Endaris**
Covers hitsound triggers and passing/failing triggers with elaborate examples.

### CODE REVIEW

File: storyboarding/scripting/compound_commands.rst [hunk 1]
```diff
@@ -12,6 +12,405 @@ Loop
...
+  For recreating the animation with triggers on its individual sprites you have to explicitly fade them out at the start of the trigger as they will otherwise fade in as soon as the trigger starts executing.
+
+.. note:: It should be noted that all combos in the map you're storyboarding for have to be at least as long as the longest transition effect (in this case 2 seconds of getting up).
+  Otherwise the effects will overlap or not flow well into each other (assuming you counteracted the overlapping issue) and make it look very bad. 2 seconds for a combo in osu! standard is very reasonable though so this would work in most beatmaps.
+
+.. You can check the result of this tutorial at <insert link here>.
+
+
+Trigger Groups
+--------------
```
Comment (cl8n) [inline storyboarding/scripting/compound_commands.rst]: if you aren't planning to write these sections for this PR, you should probably add some todo blocks down here so they aren't forgotten. same with the headers at the top

**Comment by cl8n** [comment]
awesome work! you have my approval after what I mentioned above

also, I've undone your changes to image paths. if you use backslashes, you need to escape them first, or else the compiler treats them as escape characters themselves. the rest of the project uses forward-slashes so we'll stick with those

File: storyboarding/scripting/compound_commands.rst [hunk 2]
```diff
@@ -12,6 +12,405 @@ Loop
...
-==================
+Trigger
+=======
 .. Originally called trigger-loops lol
+
+There is one mechanic within storyboarding that will allow you to create some sort of interactive effect with what is happening during gameplay: Triggers.
+Triggers can catch certain events and execute basic commands on your sprite in response.
+
+.. note:: Triggers are the most complex command currently provided by the storyboarding API and unfortunately they are accordingly buggy.
+  There is a whole batch of weird behaviour attached to the use of triggers that will be covered in its most common oddities throughout this chapter and at the end of this chapter in its less common oddities.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: This is a rather wordy run-on sentence. Either break it apart or chop it down and be more succinct. I also encourage you try using an active voice than passive, so something like:

"This section will cover triggers' most common oddities and weird behavior, along with a few edge cases towards the end."

Oh, and I guess we're also using American English spelling. But no big deal. 💩

File: storyboarding/scripting/compound_commands.rst [hunk 3]
```diff
@@ -12,6 +12,405 @@ Loop
...
+  :name: TriggerSyntax
+
+  Sprite,Background,Centre,"sb\aoba.png",192.625,140
+   F,0,23500,,0
+   S,0,23500,,0.2
+   T,HitSoundSoft,0,105330
+    F,0,0,,1
+    F,0,110,,0
+
+*T* is they character marking the triggercommand.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: they character?! 👀

Also, please space out *trigger command*.

File: storyboarding/scripting/compound_commands.rst [hunk 4]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+*HitSoundSoft* is the trigger condition. You will learn about the different kinds of trigger conditions in the next chapter.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: I kind of think of chapters as the entire "article" (or page). Perhaps section is more appropriate.

Maybe this is a lost in translation between the two of us.

File: storyboarding/scripting/compound_commands.rst [hunk 5]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+*0,105330* marks start- and endtime of the trigger. There is no limit to how often a trigger can be executed within the specified timeframe.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Be semantic and break apart the words. So it'd be "marks the start and end time of the trigger."

Feel free to also use the inline coding for ``0, 105330``. Actually, I should think about what consistent convention this should be, lol

File: storyboarding/scripting/compound_commands.rst [hunk 6]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+
+| The Fade commands after the trigger command are indented by one additional whitespace or underscore to mark them as belonging to the trigger.
+| It is very important to notice that the time values specified for these commands are **relative** to the time when the trigger starts executing.
+
+| Example: If the trigger condition is met at 23918 the sprite will change to 100% opacity at 23918 and fade out at 24028.
+| This behaviour assures that there a trigger can be activated and executed multiple times.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Remove the word *there*. It's not needed.

File: storyboarding/scripting/compound_commands.rst [hunk 7]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+Trigger Conditions
+------------------
+| Triggering a trigger is very similar to triggering a trap:
+| It will snap once someone steps into it.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: I think we can keep it as one line for this one. The first sentence isn't too long. :)

I wonder if saying "hitting a trigger" instead is less redundant, since saying the word trigger so many times kind of dulls the blow... Haha.

File: storyboarding/scripting/compound_commands.rst [hunk 8]
```diff
@@ -12,6 +12,405 @@ Loop
...
+~~~~~~~~~
+| The first big chunk of possible trigger conditions consists of hitsounds. Every time a note is hit, a hitsound will be played. But Hitsound does not equal Hitsound:
+| There are as many possible different triggers for hitsounds in a map as hitsounds available.
+
+| How do we specify the hitsound we want to have our trap snap on?
+| This is easy. The naming for the according trigger follows a simple naming scheme as shown in the following table:
+
+.. rst-class:: table table-sm table-hover table-striped
+.. csv-table:: Hitsound Trigger Conditions
+   :header: "Condition Type", "Name of Sampleset", "Additional", "Number of Sampleset", "Triggername"
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: I think I have suggestions for the following columns:

- You can shorten this to "Sampleset Name", or "Sampleset".
- Hmm... Maybe this can be the actual *hitsound*? Additional is kind of weird standing there alone. Are they actually called Additionals? Lemme know on that.
- You can shorten this to "Sampleset Number".
- Break apart "Triggername" so that it's "Trigger Name".

File: storyboarding/scripting/compound_commands.rst [hunk 9]
```diff
@@ -12,6 +12,405 @@ Loop
...
+| It is not *that* simple though. Our traps have different tastes and they are picky to different degrees.
+| Let's check how we can satisfy the taste of each of our traps.
+
+Hitsounds
+~~~~~~~~~
+| The first big chunk of possible trigger conditions consists of hitsounds. Every time a note is hit, a hitsound will be played. But Hitsound does not equal Hitsound:
+| There are as many possible different triggers for hitsounds in a map as hitsounds available.
+
+| How do we specify the hitsound we want to have our trap snap on?
+| This is easy. The naming for the according trigger follows a simple naming scheme as shown in the following table:
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst:77]: These are examples, right? We should state that they are examples so readers don't assume these are all that's available.

File: storyboarding/scripting/compound_commands.rst [hunk 10]
```diff
@@ -12,6 +12,405 @@ Loop
...
+   :widths: auto
+
+   "Hitsound", "Soft", "Whistle", "1", "HitsoundSoftWhistle1"
+   "Hitsound", "Drum", "", "", "HitsoundDrum"
+   "Hitsound", "", "Clap", "", "HitsoundClap"
+   "Hitsound", "", "", "4", "Hitsound4"
+   "Hitsound", "Normal", "", "2", "HitsoundNormal2"
+   "Hitsound", "", "Finish", "0", "HitsoundFinish0"
+
+The columns are very easy to understand when you are familiar with hitsounding. Aside from the prefix "Hitsound" you can individually add or leave out a specification of your hitsound to get the exact hitsound or combination of hitsounds you want.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: You're probably looking for "if" instead of when. Let's also switch this to active voice. "If you're familiar with hitsounding, then these columns should be very easy to understand."

File: storyboarding/scripting/compound_commands.rst [hunk 11]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+If you don't know what any of these mean, consider reading `a guide on hitsounding <https://osu.ppy.sh/s/224758>`_.
+
+ .. warning:: While the naming of the triggers suggests that it checks which hitsound is played, this is actually not the case!
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: You have extra whitespace here, causing an unintended indentation upon compiling.

File: storyboarding/scripting/compound_commands.rst [hunk 12]
```diff
@@ -12,6 +12,405 @@ Loop
...
+    Changing the sampleset of a note via the per-note quickaccess on the top left of the editor(Shift+Q/W/E/R) or via Sample Import will be ignored for the sake of evaluating triggers.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Break apart "quick access".

File: storyboarding/scripting/compound_commands.rst [hunk 13]
```diff
@@ -12,6 +12,405 @@ Loop
...
+  :align: right
+
+If you aren't familiar with Taiko mapping, no problem, we got that covered in a few words.
+
+There are 2 types of notes in Taiko, Don (red) and Kat (blue). If it has a whistle and/or clap hitsound attached it is a Kat, otherwise it is a Don note.
+
+
+We can make Aoba hit the drum by catching the respective trigger conditions:
+
+These would be ``HitsoundWhistle`` and ``HitsoundClap`` for Kat.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Let's shorten that into one line:
"by catching the respective trigger conditions: ``HitsoundWhistle`` for Don and ``HitsoundClap`` for Kat.

File: storyboarding/scripting/compound_commands.rst [hunk 14]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+For Don we have to create a workaround and give each note a separate Hitsoundset because we can't react to the hitnormal without reacting to the additionals as well.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: I cry that the first example for Triggers have to involve the workaround 😭

File: storyboarding/scripting/compound_commands.rst [hunk 15]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+So let's say we put Hitsoundset 4 for each Don, meaning we have to specify the triggercondition as ``Hitsound4``.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Break that word apart.

File: storyboarding/scripting/compound_commands.rst [hunk 16]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+
+To make this as simple as possible in terms of example, we'll introduce 2 more versions of Aoba in which she is using one or the other drumstick to hit the Taiko.
+
+We will also put one of her pigtails into a differentsprite so that we can display the drumsticks in front of the drum but her hair behind it.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Pigtails is OK, but differentsprite isn't. 😡 reacs only. Jk. <3

File: storyboarding/scripting/compound_commands.rst [hunk 17]
```diff
@@ -12,6 +12,405 @@ Loop
...
+  :name: Static sprites
+
+  Sprite,Background,Centre,"sb\aobaHair.png",192.625,140
+   S,0,0,,0.2
+   F,0,0,,1
+  Sprite,Background,CentreLeft,"sb\drum.png",186,163
+   S,0,0,,0.3
+   F,0,0,,1
+
+When nothing is happening the sprite of the idling Aoba should be shown. This will be the case at the start of the beatmap.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: With "sprite of the idling Aoba", be careful with the passive descriptions – they become a bit wordy and harder to quickly read through. Opt for "idling Aoba sprite".

File: storyboarding/scripting/compound_commands.rst [hunk 18]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+  Sprite,Background,Centre,"sb\aobaTaikoIdle.png",192.625,140
+   F,0,0,,1
+   S,0,0,,0.2
+   F,0,105330,,0
+   T,HitSound,0,105330
+    F,0,0,,0
+    F,0,110,,1
+
+Fairly simple, isn'it? Now let's do the same for the other two sprites except that they are invisible at the start and fade in on the according hitsound.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Whoa, first time I've seen that kind of fusion of "isn't it". I'm surprised it isn't widespread slang. 👀 

Let's use "corresponding" instead of "according".

File: storyboarding/scripting/compound_commands.rst [hunk 19]
```diff
@@ -12,6 +12,405 @@ Loop
...
+   T,HitSound4,0,105330
+    F,0,0,,1
+    F,0,110,,0
+
+And that's it! Aoba will now play the drum in sync with the map!
+You can see the application of this effect in the following map: https://osu.ppy.sh/s/597411
+
+.. warning:: Yeah, the map, not with the player. Hitsound-triggers are only activated by Hitsounds from objects. Otherwise this would be amazing for Taiko-mode effects but you can still do some interesting things with it!
+  The other sad thing is that spinners and drumrolls (except for the head) in Taiko don't trigger hitsounds but you could perfectly let Aoba drum to an osu!standard difficulty.
+  This solution is far from universal as you will see in the Pitfalls-section but it works perfectly fine for Kantans and most Futsuu difficulties.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Let's make the run-on sentence easier to read by adding some commas:
"From universal, as you will see in the Pitfalls section, but it"

File: storyboarding/scripting/compound_commands.rst [hunk 20]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+Change of Gamestate
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Let's break that word "Game state" apart.

File: storyboarding/scripting/compound_commands.rst [hunk 21]
```diff
@@ -12,6 +12,405 @@ Loop
...
+~~~~~~~~~~~~~~~~~~~
+.. it is safe to say that Passing/Failing triggers literally don't work in CtB and Mania
+  attention:: If you are planning to use Passing and Failing triggers in a map for either Catch the Beat or mania, go no further because these are literally not implemented.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Yeah I think this admonition is deserved. You should uncomment it, but have a bit more of a professional / serious sound to the text for importance's sake.

File: storyboarding/scripting/compound_commands.rst [hunk 22]
```diff
@@ -12,6 +12,405 @@ Loop
...
+  There is full support for standard and partial support for Taiko (only Don/Kat, no drumrolls/spinners). Keep this in mind before making big plans with this type of trigger.
+
+As you might know already from reading the chapter about layers and objects, osu! supports 4 layers of which 2 are `Pass` and `Fail` that are displayed in accordance to the current game state.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Try condensing this given the tips I've told you about prior. :)

File: storyboarding/scripting/compound_commands.rst [hunk 23]
```diff
@@ -12,6 +12,405 @@ Loop
...
+  - Passing occurs when the game state is changing from Fail to Pass
+  - Failing occurs when the game state is changing from Pass to Fail
+
+Now there is one major hiccup in applying this to storyboards and that is that Pass and Fail-Layers work differently in every game mode.
+
+In osu!standard the game state can only change at the end of a combo. If the last note of the combo gets you a Geki judgement the game state will be `Pass`, otherwise it will be `Fail`.
+
+In osu!taiko the game state can change on each note. If the last note was missed, the game state is `Fail`, otherwise it is `Pass`. It should also be noted that drumrolls (except for the head? unconfirmed) and spinners don't count as notes.
+
+In osu!mania and Catch the Beat the game state can only change on breaks and as we learned earlier, triggers always need to correspond to a gameobject. This means that `Passing` and `Failing` triggers basically don't exist for these gamemodes.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Break apart.

File: storyboarding/scripting/compound_commands.rst [hunk 24]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+
+Example: Change of Gamestate
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Yeah, you know what to do. ;)

File: storyboarding/scripting/compound_commands.rst [hunk 25]
```diff
@@ -12,6 +12,405 @@ Loop
...
+On triggering `Passing`, Aoba will get up from the ground.
+
+The implementation
+..................
+
+For the sidescrolling street we will take it easy as a start. After a quick google search a 2.5D animation of scrolling buildings turns up.
+
+Splitting that into frames, renaming the individual pictures to use as an animation (I used a script for this because it has about 250 files) and we're ready to go.
+
+.. image:: img/compound_commands/streetscroll.gif
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst:279]: Man, this would make a cool effect. 👍

File: storyboarding/scripting/compound_commands.rst [hunk 26]
```diff
@@ -12,6 +12,405 @@ Loop
...
+  :caption: Character lying on the ground
+  :name: Tripped
+
+  Sprite,Fail,Centre,"sb\falling\fallen.png",280,240
+   F,0,0,,1
+   T,Failing,0,90000
+    F,0,0,,0
+    F,0,2000,,1
+
+All that is left is putting the actual animations for falling and getting up in.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: No need for that hanging preposition.

File: storyboarding/scripting/compound_commands.rst [hunk 27]
```diff
@@ -12,6 +12,405 @@ Loop
...
+
+Yes, too easy to actually work. Otherwise this would be the coolest interactive storyboard in 29 lines ever.
+
+The problem with using animations here is that they run independently from the trigger:
+
+  - When using `LoopOnce` as the loop-option they will work fine on the first trigger but show only the last animation frame on consecutive triggers.
+  - When using `LoopForever` the animation will work fine on the first trigger but start and end on the wrong frame.
+
+
+The slightly annoying but in this case bearable workaround is animating by hand which means creating a Sprite for every frame of the animation and fade it in and out with the delay according to its position within the animation.
```
Comment (starrodkirby86) [inline storyboarding/scripting/compound_commands.rst]: Be careful with the run-ons.

"In this case, the slightly annoying but bearable workaround is animating by hand, creating a sprite for every frame..."

**Review by starrodkirby86 — CHANGES_REQUESTED** [review]
Great effort. I'm a little wary that we don't have a simplistic example, as both examples have caveats that complicate the initial example a bit more. I suppose that can come in a future update.

I think after you make the revisions I requested, I'll add this page in. I feel like at this point we can try splitting Compound Commands into Loops and Triggers individually or something, documenting all quirks for each of these.

Overall, the big thing I think you need to be careful for are your run-on sentences and your erroneous word fusions. I think these are just symptoms of the German language, lol. I recommend when correcting those things to try reading it aloud (in English), and if you're struggling to read smoothly or finding it difficult to pause, then that might be a sign to break the sentence apart or describe it more succinctly.

File: storyboarding/scripting/compound_commands.rst (hunk 8 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Changing to "Sampleset" would be ambiguous as it remains unclear whether it addresses soft/normal/drum or 1,2,3,4 without looking at the context. 
I will change that to "Sampleset Name".

As for the Additional, this derives from the editor's header for the per-note sampleset dropdown on the top left (reads "Additions"), it makes sense to change this to "Hitsound Addition". Only "Hitsound" is too inaccurate of a term to use because it is not possible to specify hitnormals as an own type of hitsound here.

Comment (starrodkirby86) [reply storyboarding/scripting/compound_commands.rst]: Could we opt for Hitsound Addition, then?

File: storyboarding/scripting/compound_commands.rst (hunk 2 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Stole your sentence!

File: storyboarding/scripting/compound_commands.rst (hunk 3 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: fixed

File: storyboarding/scripting/compound_commands.rst (hunk 4 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: I agree that chapter is too big of a word. Section sounds awkard for me as it has a wide definition but I don't have a better idea. I will go with section for now.

File: storyboarding/scripting/compound_commands.rst (hunk 5 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Followed your suggestion for wording. As for the inline coding, it would be consequent to do it for the other to fragments taken from the code as well.
I changed this to ``stuff`` for now as I'm mostly using that in the later parts as well.

File: storyboarding/scripting/compound_commands.rst (hunk 6 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: done

File: storyboarding/scripting/compound_commands.rst (hunk 7 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Linebreak removed.
Hitting is a meh word imo as its characteristics envelop activity and intention while triggers just...get triggered. Kept "triggering a trigger".

File: storyboarding/scripting/compound_commands.rst (hunk 9 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst:77]: Hm, personally I think the sentence "There are as many possible different triggers for hitsounds in a map as hitsounds available." makes this quite clear.
Anyway, I changed the title of the table to "Selection of Hitsound Trigger Conditions". If you think further changes are necessary to underline this, please take action yourself :>

File: storyboarding/scripting/compound_commands.rst (hunk 8 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Yes, done.

File: storyboarding/scripting/compound_commands.rst (hunk 10 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: screw german.
if = wenn
The lame thing is that on several occasions "when" and "if" are sort of interchangeable (line 99 for example) which makes it too temptating to use "when" due to the phonetic similarity. 
Fixed.

File: storyboarding/scripting/compound_commands.rst (hunk 11 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Fixed.

File: storyboarding/scripting/compound_commands.rst (hunk 12 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Changed "per-note quickaccess" to "Sampleset Menu" because the former is really a stupid and indescriptive term, LOL.
I also got hella confused at "on the top left of the editor" which sounds wrong on rereading it but as you didn't comment on it, I just left it in the end.

File: storyboarding/scripting/compound_commands.rst (hunk 13 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Eeeh. I only saw on second sight that this change does not make sense, otherwise I would have commented earlier. 
I see that the connection from the : to the next sentence is iffy but ``HitsoundWhistle`` is not for Don and only pulling Kat into that sentence seems inconsequent.
I did a fair bit of rewording in this part to make it less awkward.

File: storyboarding/scripting/compound_commands.rst (hunk 14 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: While I agree that this should usually be avoided, the frustration and tears on my own journey  of getting triggered led me to believe that it is most important to know what does `not` work with triggers and how you can circumvent it rather than having some theoretical knowledge that runs you into a wall after you finished your conception.

File: storyboarding/scripting/compound_commands.rst (hunk 15 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Ok.

File: storyboarding/scripting/compound_commands.rst (hunk 16 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Fixed.

File: storyboarding/scripting/compound_commands.rst (hunk 17 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Changed. Although I have to note that it sounds more cute when it's a sprite of Aoba instead of an Aoba sprite. More personal involvement from Aoba, you know ;)

File: storyboarding/scripting/compound_commands.rst (hunk 18 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: I really don't know either, I thought it was common practice to merge words through the use of ' in order to shorten your sentence. Weird brits and muricans.
Followed your suggestion.

File: storyboarding/scripting/compound_commands.rst (hunk 19 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: ok

File: storyboarding/scripting/compound_commands.rst (hunk 20 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Done.

File: storyboarding/scripting/compound_commands.rst (hunk 21 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Oh yes, I forgot that I wanted to change this.

File: storyboarding/scripting/compound_commands.rst (hunk 22 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Back in english class >.<"
Should be better now, please be gentle with grading, Kirby-sensei.

File: storyboarding/scripting/compound_commands.rst (hunk 23 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: - and as
+. As

File: storyboarding/scripting/compound_commands.rst (hunk 24 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Yes.

File: storyboarding/scripting/compound_commands.rst (hunk 25 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst:279]: Wikipedia article covering 2.5D, lmao
I agree.

File: storyboarding/scripting/compound_commands.rst (hunk 26 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Uh oh, what was a hanging preposition again?
Reconstructed somehow.

File: storyboarding/scripting/compound_commands.rst (hunk 27 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst]: Fixed.

File: storyboarding/scripting/compound_commands.rst (hunk 9 above)
Comment (starrodkirby86) [reply storyboarding/scripting/compound_commands.rst:77]: Wow, you just don't want me to make another review, huh? :p

File: storyboarding/scripting/compound_commands.rst (hunk 14 above)
Comment (starrodkirby86) [reply storyboarding/scripting/compound_commands.rst]: I think in a future version of the article (maybe when it's split into its own page), we can add a basic example first, then have the next two examples be two scenarios demonstrating triggers' usefulness and shortcomings.

File: storyboarding/scripting/compound_commands.rst (hunk 9 above)
Comment (Endaris) [reply storyboarding/scripting/compound_commands.rst:77]: Another review is fine by me but in this particular case we're talking about actual content while the rest was mostly about wording and formatting. To my eye the message is transmitted in sufficient clarity with the latest addition based on your feedback so you should have a very specific idea on how that should be communicated with the reader instead if you don't share that viewpoint. And if you already have that idea, I'm sure you can put that into reality better than I can. If I have objections to that change I can still voice that after and it's probably less trouble for both of us.
